internal/telemetry/cluster.go

```diff
@@ -202,6 +203,19 @@ func (c *Collector) IsPlusEnabled() bool {
 	return c.Config.IsPlus
 }
 
+// InstallationFlags returns the list of all flags
+func (c *Collector) InstallationFlags() []string {
+	flags := os.Args[1:]
+	// tests that compare the report currently break with this addition,
```

Comment:
In this case this PR may be a `draft PR` with a notice: DO NOT MERGE?

Reply:
I think what @AlexFenlon is saying here is not that tests break now, but that adding the flags to telemetry, adds the flags also to the struct that is compared when running go tests. I agree it would be better to ignore that flag on the go test side though, than in the go code. I don't think there is a good way to test this feature properly on the go tests side.

Reply:
Args passed to the NIC is a slice - flags. In tests we need to simulate arguments passed to the program. `FlagSet` is one of the options: https://pkg.go.dev/flag#FlagSet. As always, it's handy to ask: What are we really testing here?